asab/zookeeper/container.py

```diff
@@ -94,6 +94,7 @@ async def _do_advertise(self, zoocontainer):
 				await zoocontainer.ZooKeeper.set_data(self.Node, self.Data)
 				return
 
+			await zoocontainer.ZooKeeper.ensure_path(self.Path.rstrip(self.Path.split("/")[-1]))
```

Comment:
1) You cannot simply expect that only a last part of the path needs to be created
2) This should be done only in the first go (use flag for that) or as a reaction to the exception

Reply:
It is not the last part. It is the path BUT the last part because the last part here is the application name. Handling the exception was what I did in the first attempt. I'm lost...

Reply:
Ok, let's discuss this then after planning.